managing_providers/_topics/embedded_workflows.md

```diff
@@ -187,11 +187,26 @@ Workflows must be authored in Amazon State Languages (ASL) format. As part of au
       
       You are recommended to use a docker.io [access token](https://docs.docker.com/security/for-developers/access-tokens/) so that the token does not expire.
 
-   * For OCP provide an image pull secret to a podified Kubernetes container, and then add it to a service  account
+   * Provide an image pull secret to a podified Kubernetes container, and then add it to a service account
 
      In order to pull an image from a private registry you have to provide an `ImagePullSecret` to your containers, see [Pull an Image from a Private Registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). {{ site.data.product.title_short }} uses a service account called {{ site.data.product.workflow_service_account }} to run containers for your workflows. You can add an `ImagePullSecret` to this service account by following [Add Image Pull Secrets to a service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account/).
 
-    
+#### Running an Embedded Workflow on the Openshift Container Platform (OCP) 
```

Comment:
Should there be a `#### Running an Embedded Workflow an Appliances` section above to match this?